From cd9bb710e7d6b23188ac354b3fec0246aa5f9c5c Mon Sep 17 00:00:00 2001
From: Yong-iL Joh <yong-il.joh@windriver.com>
Date: Tue, 17 Nov 2015 16:28:11 +0900
Subject: [PATCH] ivi-shell-user-interface: fix runtime error caused by missing
 header inclusion

when weston is running on qemux86 device, there is an error with following.

[08:02:07.897] launching '/usr/lib/weston/weston-ivi-shell-user-interface'
[08:02:08.201] /usr/lib/weston/weston-ivi-shell-user-interface died on signal 11

this is caused by type mismatch, and
it does occur on qemux86-64

Signed-off-by: Yong-iL Joh <yong-il.joh@windriver.com>
Acked-by: Daniel Stone <daniel@fooishbar.org>
Reviewed-by: Bryce Harrington <bryce@osg.samsung.com>
---
 clients/ivi-shell-user-interface.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/clients/ivi-shell-user-interface.c b/clients/ivi-shell-user-interface.c
index 0bf2723..460ca27 100644
--- a/clients/ivi-shell-user-interface.c
+++ b/clients/ivi-shell-user-interface.c
@@ -21,6 +21,8 @@
  * DEALINGS IN THE SOFTWARE.
  */
 
+#include "config.h"
+
 #include <sys/wait.h>
 #include <unistd.h>
 #include <stdlib.h>
-- 
1.9.1

